### ISSUE TITLE
Sites Tree Attack Menu grayed out when selecting folder/single URL at same level

### ISSUE THREAD

**Issue by davewichers** [zaproxy/zaproxy#4546]
If my Sites Tree has a single URL and then a folder at the same level in the tree right before or after that URL and I multiselect both and then Right-Click to select the Attack menu, all the items in that menu are grayed out. I don't see why they should be grayed out. I should be able to multi-select any non-overlapping entries in the sites tree and be able to spider/scan/whatever them.

I can see it being grayed out if you multiselect both a folder and some of the entries inside the same folder.

**Comment by thc202** [comment]
Do you want to scan both the folder and the URL then?

The menus are disabled because the scans expect just one starting point.

**Comment by davewichers** [comment]
Yes. I want all the selected items to be scanned/attacked. Whatever the user has selected.
